kafka-connect/kafka-connect/src/main/java/org/apache/iceberg/connect/CommitterFactory.java

```diff
@@ -21,7 +21,7 @@
 import org.apache.iceberg.connect.channel.CommitterImpl;
 
 class CommitterFactory {
-  static Committer createCommitter(IcebergSinkConfig config) {
+  static Committer createCommitter() {
```

Comment:
We should leave this as-is for now and address changes in the PR to support different committers.